requests_oauthlib/oauth2_session.py

```diff
@@ -246,8 +246,14 @@ def refresh_token(self, token_url, refresh_token=None, body='', auth=None,
         body = self._client.prepare_refresh_body(body=body,
                 refresh_token=refresh_token, scope=self.scope, **kwargs)
         log.debug('Prepared refresh token request body %s', body)
+
+        headers = headers or {
+            'Accept': 'application/json',
+            'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8',
+        }
```

Comment:
Can I recommend changing this logic so it only overrides the headers is `headers` is `None`? It would be useful to allow people to explicit request no additional headers.